src/browser/NativeMessageInstaller.h

```diff
@@ -42,6 +42,8 @@ class NativeMessageInstaller
     QJsonObject constructFile(BrowserShared::SupportedBrowsers browser);
     bool createNativeMessageFile(BrowserShared::SupportedBrowsers browser);
 
+    QString constructFlatpakPath() const;
+
```

Comment:
Either ifdef this, too, or make it a free function without a declaration in the header file.

Reply:
done